.github/dependabot.yml

```diff
@@ -1,20 +0,0 @@
-# Please see the documentation for all configuration options:
-# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-
-version: 2
-updates:
-  - package-ecosystem: "npm" # this section is for version update PRs
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "daily"
-    allow:
-      - dependency-name: "ganache"
-    reviewers:
-      - "trufflesuite/truffle-core-devs"
-  - package-ecosystem: "npm" # this section is for security update PRs
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "weekly"
-    # Disable version updates for npm dependencies
-    # see https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#overriding-the-default-behavior-with-a-configuration-file
-    open-pull-requests-limit: 0
```

Comment:
@davidmurdoch what happens if this setting is removed?

Reply:
The problem with the current config is that github doesn't allow us to specify `npm` twice. You are looking at the 2nd of 2 npm configs. This `open-pull-requests-limit: 0` setting is the official documented ~hack~ way to enable security-only updates; setting `open-pull-requests-limit` _actually_ sets it to `10` if the Security Updates setting is enabled in the GitHub UI.